strawberry/schema/schema_converter.py

```diff
@@ -103,26 +120,40 @@ def from_enum(self, enum: EnumDefinition) -> CustomGraphQLEnumType:
 
         return graphql_enum
 
-    def from_enum_value(self, enum_value: EnumValue) -> GraphQLEnumValue:
-        return GraphQLEnumValue(enum_value.value)
+    def from_enum_value(
+        self, enum_value: EnumValue, parent_docstring: Optional[Docstring] = None
+    ) -> GraphQLEnumValue:
+        description = enum_value.description
+        if description is None and self.config.description_from_docstrings:
+            description = docstrings.attribute_description(
+                parent_docstring, enum_value.name
+            )
+        return GraphQLEnumValue(enum_value.value, description=description)
 
     def from_directive(self, directive: StrawberryDirective) -> GraphQLDirective:
+        description = directive.description
+        docstring = directive.docstring
+        if description is None and self.config.description_from_docstrings:
+            description = docstrings.type_description(docstring)
```

Comment:
shall we make a a `self._get_description` utility method? :)

Reply:
Good idea

Reply:
Done